components/ftx/browser/ftx_json_parser.cc

```diff
@@ -0,0 +1,285 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <string>
+
+#include "brave/components/ftx/browser/ftx_json_parser.h"
+
+#include "base/json/json_reader.h"
+#include "base/logging.h"
+#include "base/strings/string_number_conversions.h"
+#include "base/values.h"
+
+bool FTXJSONParser::GetFuturesDataFromJSON(const std::string& json,
```

Comment:
this doesn't necessarily need to happen in this PR, but we should be using JSONValueConverter for this kind of thing